lib/view_component/test_helpers.rb

```diff
@@ -21,9 +21,11 @@ def refute_component_rendered
       # :nocov:
     end
 
-    attr_reader :rendered_component
+    attr_reader :controller, :variant, :rendered_component
```

Comment:
In this new factoring is `controller` used outside of the `render_inline` method? If not why not make it just a variable in that method?